plugins/engines/ospray/OSPRayModel.cpp

```diff
@@ -0,0 +1,414 @@
+/* Copyright (c) 2015-2018, EPFL/Blue Brain Project
+ * All rights reserved. Do not distribute without permission.
+ * Responsible Author: Cyrille Favreau <cyrille.favreau@epfl.ch>
+ *
+ * This file is part of Brayns <https://github.com/BlueBrain/Brayns>
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License version 3.0 as published
+ * by the Free Software Foundation.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this library; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+#include "OSPRayModel.h"
+#include "OSPRayMaterial.h"
+
+#include <brayns/common/material/Material.h>
+namespace
+{
+const size_t BOUNDINGBOX_MATERIAL_ID = 9999;
```

Comment:
no more than 10000-2 materials? so sad :package:

Reply:
I regularly use more than that actually. Maybe set it to size_t (-1)?

Reply:
I like that! Done.